core/domain/app_feedback_report_services.py

```diff
@@ -136,32 +136,32 @@ def _update_report_stats_model_in_transaction( # type: ignore[no-untyped-def]
             stats_id))
 
     stats_parameter_names = (
-        app_feedback_report_constants.STATS_PARAMETER_NAMES)
+        app_feedback_report_constants.StatsParameterNames)
     if stats_model is None:
         assert delta > 0
         # Create new stats model entity. These are the individual report fields
         # that we will want to splice aggregate stats by and they will each have
         # a count of 1 since this is the first report added for this entity.
         stats_dict = {
-            stats_parameter_names.report_type.name: {
+            stats_parameter_names.report_type.value: {
```

Comment:
You changed the key `report_type` to `REPORT_TYPE` in the 'StatsParameterNames' enum, but you forgot to change it here.

It should be `stats_parameter_names.REPORT_TYPE.value`. (ditto for the lines below).